packages/application-shell/src/components/theme-switcher/theme-switcher.tsx

```diff
@@ -0,0 +1,10 @@
+import { ThemeProvider } from '@commercetools-uikit/design-system';
+import { useFeatureToggle } from '@flopflip/react-broadcast';
+import { UI_REDESIGN } from '../../feature-toggles';
+
+const ThemeSwitcher = () => {
```

Comment:
**nit**: I would expect a component with this name to actually implement a theme selector. Perhaps we will have it in the future when we implement dark theme.

Reply:
Yeah we will need to refactor things again once we get there.